src/main/scala/firrtl/analyses/InstanceGraph.scala

```diff
@@ -70,14 +61,14 @@ class InstanceGraph(c: Circuit) {
   /** A list of absolute paths (each represented by a Seq of instances)
     * of all module instances in the Circuit.
     */
-  lazy val fullHierarchy = graph.pathsInDAG(trueTopInstance)
+  lazy val fullHierarchy: collection.Map[WDefInstance,Seq[Seq[WDefInstance]]] = graph.pathsInDAG(trueTopInstance)
 
   /** Finds the absolute paths (each represented by a Seq of instances
     * representing the chain of hierarchy) of all instances of a
     * particular module.
     *
     * @param module the name of the selected module
-    * @return a Seq[Seq[WDefInstance]] of absolute instance paths
+    * @return a Seq[ Seq[WDefInstance] ] of absolute instance paths
```

Comment:
Curiosity: scaladoc doesn't like not having the spaces?

Reply:
So double square-brackets indicate a type in scaladoc, so the end of this type would be recognized as a weird trailing bracket. I put spaces in to just avoid it messing up. I'm sure the correct fix.